components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -482,4 +510,35 @@ private void invalidatePendingEmailVerification(User user, UserStoreManager user
             }
         }
     }
+
+    /**
+     * Logs the message to the audit log file
```

Comment:
lets add a full stop

Reply:
Fixed in https://github.com/wso2-extensions/identity-governance/pull/452/commits/db3db432beb65c5e6174862949a01252d2f26d94